OpenRA.Mods.TS/UtilityCommands/LegacyTilesetImporter.cs

```diff
@@ -137,6 +157,41 @@ public void Run(ModData modData, string[] args)
 			{
 				// GetSection will throw when we run out of sections to import
 			}
+
+			metadata.Append("\tEditorTemplateOrder: ");
+
+			categories = categories.Distinct().ToList();
+			metadata.Append(categories.First());
+			foreach (var category in categories.Skip(1))
```

Comment:
This is overkill. Something like `metadata.Append("\tEditorTemplateOrder: " + string.Join(", ", categories.Distinct()));` should work fine.